### PR TITLE
get the CurrentBranch from git instead of parsing text

### PR DESCRIPTION
**Pull request by 1ace**

### CODE REVIEW

File: internal/git/git.go [hunk 1]
```diff
@@ -110,29 +110,14 @@ func Log(sha1, sha2 string) (string, error) {
 	return string(outputs), nil
 }
 
-// CurrentBranch returns the currently checked out branch and strips away all
-// but the branchname itself.
+// CurrentBranch returns the currently checked out branch
 func CurrentBranch() (string, error) {
-	cmd := New("branch")
-	cmd.Stdout = nil
```
Comment (zaquestion) [inline internal/git/git.go]: Error in CI is because you still need this line I'm afraid. Great looking changes tho! Love deleted code!

**Comment by zaquestion** [comment]
Closing in favor of https://github.com/zaquestion/lab/pull/324 to bring in these changes

**Comment by 1ace** [comment]
Hey, thanks for fixing this, and sorry for forgetting about this PR :man_facepalming:

**Comment by zaquestion** [comment]
You're welcome, no worries, I would have poked you again, but was just on a roll at GopherCon joyfully catching up on the open PRs :)

**Pull request closed without merging**